appveyor.yml

```diff
@@ -1,22 +1,58 @@
 version: 1.0.{build}
+os:
+- Visual Studio 2013
+# only run PR-CA on the VS 2015 OS - please change the scripts if this string changes
+- Visual Studio 2015
+
+install:
+  - git submodule update --init --recursive
+  - ps: |
+      Add-Type -AssemblyName System.IO.Compression.FileSystem
+      if (!(Test-Path -Path "C:\maven" )) {
+        (new-object System.Net.WebClient).DownloadFile(
+          'http://www.us.apache.org/dist/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.zip',
+          'C:\maven-bin.zip'
+        )
+        [System.IO.Compression.ZipFile]::ExtractToDirectory("C:\maven-bin.zip", "C:\maven")
+      }
+  - cmd: SET PATH=C:\maven\apache-maven-3.2.5\bin;%JAVA_HOME%\bin;%PATH%
+
+cache:
+  - C:\sonarqube\downloads -> appveyor.yml
+  - C:\users\appveyor\.m2 -> appveyor.yml
+
+configuration: Release
 
 environment:
   matrix:
-  - Configuration: Release
-  - Configuration: Debug
```

Comment:
It would be too expensive to run both configs (we're already running the tests 4 times - 2 x OS and 2 x SQ_Version). Let me know if you think there is value in running both configs for PR-CA